core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -112,10 +112,8 @@ object BattleDamage {
         if (attacker is MapUnitCombatant) {
             modifiers.add(getTileSpecificModifiers(attacker, defender.getTile()))
 
-            // Depreciated Version
-            if (attacker.unit.isEmbarked() && !attacker.unit.hasUnique(UniqueType.AttackFromSea))
-                modifiers["Landing"] = -50
-            if (attacker.unit.isEmbarked() && !attacker.unit.hasUnique(UniqueType.AttackAcrossCoast))
+            if (attacker.unit.isEmbarked()
+                    && !(attacker.unit.hasUnique(UniqueType.AttackAcrossCoast)) || attacker.unit.hasUnique(UniqueType.AttackFromSea))
```

Comment:
Usually, I prefer the opposite version: it take only 2 operations instead of 3 - less calculations, and less operations to keep in mind while reading.
Probably, that's a matter of taste.

Reply:
P.S. Your suggested example contains 1 extra ) and will not compile. 😜